### PR TITLE
[ID-1279] Log requested and actual cloud in Bard

### PR DESCRIPTION
**Pull request by samanehsan**
**Ticket:** https://broadworkbench.atlassian.net/browse/ID-1279

**What:**
We want to know when data is being egressed out of Azure. Because Azure does not have a requester pays feature like GCP, whoever is hosting/paying for storage will also need to pay any egress costs.  

**How:**
When the resolve endpoint gets called, if the response contains an `accessUrl` include the `resolvedCloud` in the Bard event properties if it is either azure or gcp.

### CODE REVIEW

File: service/src/main/java/bio/terra/drshub/services/DrsResolutionService.java [hunk 1]
```diff
@@ -42,6 +42,7 @@ public class DrsResolutionService {
   private final DrsApiFactory drsApiFactory;
   private final DrsProviderService drsProviderService;
   private final AuthService authService;
+
```
Comment (tlangs) [inline service/src/main/java/bio/terra/drshub/services/DrsResolutionService.java]: Is there a reason there's a newline here?

File: service/src/test/java/bio/terra/drshub/controllers/DrsHubApiControllerTest.java [hunk 2]
```diff
@@ -120,7 +120,7 @@ void testFallbackWhenOnlyAccessUrlRequestedWithPassportsHasEmptyPassport() throw
     mockExternalCredsGetProviderAccessToken(
         Provider.fromValue(cidProviderHost.drsProvider().getEcmFenceProvider().get().getUriValue()),
         TEST_ACCESS_TOKEN,
-            TEST_FENCE_SA_TOKEN);
+        TEST_FENCE_SA_TOKEN);
```
Comment (tlangs) [inline service/src/test/java/bio/terra/drshub/controllers/DrsHubApiControllerTest.java:123]: Where did these whitespace changes come from?

**Review by tlangs — APPROVED** [review]
Just some whitespace nits.

File: service/src/test/java/bio/terra/drshub/controllers/DrsHubApiControllerTest.java (hunk 2 above)
Comment (Shakespeared) [reply service/src/test/java/bio/terra/drshub/controllers/DrsHubApiControllerTest.java:123]: My drshub PR also created these same whitespace changes 🤔

**Review by Shakespeared — APPROVED** [review]

**Pull request merged**